JH5/task10/src/homework/Ping.java

```diff
@@ -0,0 +1,61 @@
+package homework;
+
+import java.util.*;
+
+/**
+ * Class Ping emulates request of ip addresses and write down random time of requests
+ * to the collection of ip addresses
+ */
+public class Ping {
+
+    /**
+     * Collection of ip addresses
+     */
+    private HashMap<String, Integer> pingAddresses = new HashMap<>();
+
+    /**
+     * Regular expression to verify correct ip address
+     */
+    private String regExIpV4 = "^([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\." +
+            "([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\." +
+            "([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\." +
+            "([01]?\\d\\d?|2[0-4]\\d|25[0-5])$";
+
+    /**
+     * Constructor Ping gets collection of ip addresses and complements it
+     * with random time of emulating request
+     *
+     * @param ipAddresses collection of ip addresses
+     */
+    public Ping(HashMap<String, Integer> ipAddresses) throws Exception {
```

Comment:
what if somebody wants to to ping only one address? Please, make implementation relate not so tight to other parts of your application.